base/timing.jl

```diff
@@ -57,7 +57,7 @@ function GC_Diff(new::GC_Num, old::GC_Num)
                    new.poolalloc    - old.poolalloc,
                    new.bigalloc     - old.bigalloc,
                    new.freecall     - old.freecall,
-                   new.total_time   - old.total_time,
+                   (new.total_time  - old.total_time) + (new.total_time_to_safepoint - old.total_time_to_safepoint),
```

Comment:
I would prefer introducing a new field in GC_Diff that is the safepoint time, and break the top-level report into two sections. As an example when I run a scaling test, I want to see that `total_time` stays constant (e.g. the work of GC scales well with the number of threads and GC workers). If gc time starts creeping up I might use that as a decision point to use more GC threads.

On the other hand if the latency to start GC is high (time to safepoint) I can use that as a decision point to look with a profiler if I spend to much time in certain ccalls.

Munging them together yields a simple: "GC is slow" but no actionable information.

Reply:
Yeah, that's fair. In my view, "GC is slow" is very actionable information — and throwing more information at people isn't always helpful.  How would you want this to appear?  Perhaps we take inspiration from compilation time?

> `2.39% compilation time: 12% of which was recompilation`